app/styles/custom-components/_vlc-agenda-items.scss

```diff
@@ -199,3 +192,52 @@ $i: 0;
   opacity: 1;
   color: $napier-green;
 }
+
+.vlc-agenda-items__body {
+  align-items: center;
+  padding: 1.2rem 1.8rem;
+
+  .vlc-agenda-items__body-padding {
+    padding-bottom: 1.5rem;
+  }
+}
+
+.vlc-agenda-items__remarks {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  width: 100%;
+}
+
+.vlc-agenda-items__icons {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+
+  .vlc-agenda-items__icons::before {
+    color: #FFF;
+  }
+}
+
+.vlc-agenda-items__confidentialityButton span {
+  color: #FFFFFF;
+  margin-right: 0.1rem;
+  align-items: center;
+}
+
+.vlc-agenda-items__body--flex {
```

Comment:
It's strange to use a modifier for the flex. A modifier should be used as a modification of something that already exists on its own. Like a button, can have a modifier `--bigger` or `--red`. But using flex feels more like a utility here, but I could be wrong?